### DISCUSSION TITLE
按钮样式的多选框无法使用

### DISCUSSION THREAD

**Discussion by SambacFeng** [TaleLin/lin-ui#1295] [求助] [answered]
![QQ截图20210203005935](https://user-images.githubusercontent.com/56753082/106634841-25a82b80-65bb-11eb-989e-caca053eeacb.png)
示例小程序的表单-checkbox中这部分按钮样式的多选框无法使用，在开发者工具中调试，点击按钮后也并没有触发bind:linchange事件

**Comment by juzi214032** [comment, marked as answer]
感谢反馈，会尽快修复
